R/training_window.R

```diff
@@ -0,0 +1,109 @@
+#' Limits the size of the training window to the most recent observations
+#'
+#' `step_training_window` creates a *specification* of a recipe step that
+#'   limit the size of the training window to the `nrec` most recent
+#'   observations in `time_value` per location from `geo_value`.
+#'
+#' @param recipe A recipe object. The step will be added to the
+#'  sequence of operations for this recipe.
+#' @param ... One or more selector functions to choose variables
+#'  for this step. See [selections()] for more details.
+#' @param role For model terms created by this step, what analysis role should
+#'  they be assigned?
+#' @param trained A logical to indicate if the quantities for
+#'  preprocessing have been estimated.
+#' @param nrec An integer value that represents the number of most recent
+#' observations that are to be kept in the training window per location.
+#' The default value is 50.
+#' @param skip A logical. Should the step be skipped when the
```

Comment:
The step should always be skipped on the test data.

Reply:
Right... So I've just removed `skip` from the user-facing parameters and that documentation for it.